src/Table/Table.tsx

```diff
@@ -36,6 +36,7 @@ export interface Props {
   initialSortState?: SortState;
   onPageChange?: Function;
   onRowClick?: Function;
+  onRowMouseOver?: Function;
```

Comment:
Can we use a more specific type than `Function`? I see we're using `Function` a lot in this file, but it hides a lot of type information we could potentially have since all it indicates is that `onRowMouseOver` is a function and nothing about its arguments or return type. Even just something like `(e: any, rowID: any, rowData: any): void` is a lot better since it tells us the number of arguments we expect and names that indicate what they're for, which is helpful both for typechecking and for a user trying to figure out what arguments it takes.